packages/core/cli/src/scripts/createProject/index.ts

```diff
@@ -6,24 +6,30 @@ import * as process from 'process';
 import * as fs from 'fs';
 
 interface ICreateProjectProps {
-  integration: string,
-  targetPath: string,
-  repositoryLink: string
+  integration: string;
+  targetPath: string;
+  repositoryLink: string;
 }
 
 async function createProject({
   integration,
   targetPath,
   repositoryLink
 }: ICreateProjectProps): Promise<void> {
-  const templatePath = `${targetPath}/${integration}`;
-  try {
-    if (fs.existsSync(templatePath)) {
-      fs.rmdirSync(targetPath, { recursive: true });
+  const templatePath = path.join(targetPath, integration);
+
+  if (fs.existsSync(templatePath)) {
+    try {
+      await shell.exec(`rimraf ${templatePath}`);
```

Comment:
Try directly calling `rimraf` instead of through `shell`.

Reply:
I will try but for now, everything works as expected and we need to ship a new version of cli as fast as possible.